include/mod_inbox.hrl

```diff
@@ -1,15 +1,29 @@
 -type username() :: binary().
```

Comment:
I know it's not you @fenek who introduced the type. It seems to me this type is the same as `jid:luser/0`